USAGE.md

```diff
@@ -47,7 +47,7 @@ Here is a list of files generated:
 
 | File                        | Location     | Description |
 |-----------------------------|--------------|------------|
-| Packwerk configuration      | packwerk.yml | See [Setting up configuration file](#Setting-up-configuration-file) |
+| Packwerk configuration      | packwerk.yml | See [Setting up configuration file](#Setting-up-the-configuration-file) |
```

Comment:
Let's also rephrase this to "Setting up the configuration file"? In other words there is missing "the"